### PR TITLE
[PERFSCALE-2513] Update the timeout

### PR DESCRIPTION
**Pull request by krishvoor**
This PR updates the timeout across the jobs

### CODE REVIEW

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly-x86.yaml [hunk 1]
```diff
@@ -68,6 +68,7 @@ tests:
     test:
     - chain: openshift-qe-node-density-heavy
     workflow: rosa-aws-sts
+  timeout: 5h0m0s
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly-x86.yaml:71]: for the individual tests like node-density-heavy and node-density-cni I think this timeout is a bit high, but we can keep it as is to keep consistent

Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly-x86.yaml:71]: +1

Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly-x86.yaml:71]: Agreed, keeping this as "constant" across 24nodes/small-scale testing.

**Comment by paigerube14** [comment]
looks good to me!

**Comment by krishvoor** [comment]
> looks good to me!

Thanks @paigerube14 , can you give here a `/lgtm` plz?

**Pull request merged**